userspace/libsinsp/sinsp_filtercheck_static.cpp

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2025 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <libsinsp/sinsp_filtercheck_static.h>
+
+using namespace std;
+sinsp_filter_check_static::sinsp_filter_check_static() {
+	static const filter_check_info s_no_static_fields_info = {
+	        "static",
+	        "",
+	        "Static, user-defined filters",
+	        0,
+	        nullptr,
+	        filter_check_info::FL_NONE,
+	};
+	m_info = &s_no_static_fields_info;
+}
+
+sinsp_filter_check_static::sinsp_filter_check_static(
+        const std::unordered_map<std::string, std::string> filters) {
+	m_filters = filters;
+
+	m_info_fields.resize(m_filters.size());
+
+	int i = 0;
+	for(const auto& f : m_filters) {
+		// String only supported
+		m_info_fields[i].m_type = ppm_param_type::PT_CHARBUF;
+		// Automatically suggest it for output print format
+		m_info_fields[i].m_flags = EPF_FORMAT_SUGGESTED;
```

Comment:
We mark the field infos as FORMAT SUGGESTED so that, in Falco with `append_output.suggested_output` enabled in config, the static fields are automatically attached to the output.